src/Sulu/Bundle/AutomationBundle/Controller/TaskController.php

```diff
@@ -0,0 +1,203 @@
+     *
+     * @param int $id
+     *
+     * @return Response
+     */
+    public function getAction($id)
+    {
+        $manager = $this->getTaskManager();
+        $task = $manager->findById($id);
+
+        if (!$task) {
+            throw new NotFoundHttpException();
+        }
+
+        return $this->handleView($this->view($task));
+    }
+
+    /**
+     * Create new task.
+     *
+     * @param Request $request
+     *
+     * @return Response
+     */
+    public function postAction(Request $request)
+    {
+        $manager = $this->getTaskManager();
+        $task = $this->get('serializer')->deserialize(
+            json_encode($request->request->all()),
+            Task::class,
+            'json',
+            DeserializationContext::create()->setGroups(['api'])
+        );
+        $task = $manager->create($task);
+
+        $this->getEntityManager()->flush();
+
+        return $this->handleView($this->view($task));
+    }
+
+    /**
+     * Update task with given id.
+     *
+     * @param int $id
+     * @param Request $request
+     *
+     * @return Response
+     */
+    public function putAction($id, Request $request)
+    {
+        $task = $this->get('serializer')->deserialize(
+            json_encode(array_merge(['id' => $id], $request->request->all())),
+            Task::class,
+            'json',
+            DeserializationContext::create()->setGroups(['api'])
+        );
+
+        $manager = $this->getTaskManager();
+        $manager->update($task);
+
+        $this->getEntityManager()->flush();
+
+        return $this->handleView($this->view($task));
+    }
+
+    /**
+     * Removes task with given id.
+     *
+     * @param int $id
+     *
+     * @return Response
+     */
+    public function deleteAction($id)
+    {
+        $manager = $this->getTaskManager();
+        $manager->remove($id);
+
+        $this->getEntityManager()->flush();
+
+        return $this->handleView($this->view());
+    }
+
+    /**
+     * Removes multiple tasks identified by ids parameter.
+     *
+     * @param Request $request
+     *
+     * @return Response
+     */
+    public function cdeleteAction(Request $request)
+    {
+        $manager = $this->getTaskManager();
+
+        $ids = array_filter(explode(',', $request->get('ids')));
+        foreach ($ids as $id) {
+            $manager->remove($id);
+        }
+
+        $this->getEntityManager()->flush();
+
+        return $this->handleView($this->view());
+    }
+
+    /**
+     * Returns field-descriptors for task-entity.
+     *
+     * @return FieldDescriptorInterface[]
+     */
+    private function getFieldDescriptors()
+    {
+        return $this->get('sulu_core.list_builder.field_descriptor_factory')->getFieldDescriptorForClass(Task::class);
+    }
+
+    /**
+     * Returns task-manager.
+     *
+     * @return TaskManagerInterface
+     */
+    private function getTaskManager()
+    {
+        return $this->get('sulu_automation.tasks.manager');
```

Comment:
i have shortly discussed this with @chirimoya and we will go wi this way. maybe refactoring in a later PR